composer.json

```diff
@@ -7,7 +7,7 @@
     "require": {
         "php": "^8.1",
         "phpstan/phpstan": "^1.10.26",
-        "nette/utils": "^3.2",
+        "nette/utils": "^3.2 || ^4.0",
```

Comment:
This should not be needed for downgraded install. It must pass without any package dependencies.

Reply:
ohh good point.

Reply:
hmm but composer might not let you install the package because it cannot resolve dependencies..?

Reply:
That should not happen.
Could you provide falling CI job? I'll check it

Reply:
You are right. It [works in CI](https://github.com/TomasVotruba/unused-public/pull/85) .. not sure why it didn‘t locally

Reply:
Composer sometimes get glitches with these non-standard tagging. The best way is to remove composer.lock and `/vendor`.